### PR TITLE
verify shown only for open markets

### PR DESCRIPTION
**Pull request by Mi-Lan**

### CODE REVIEW

**Comment by KadenZipfel** [comment]
Looks like the verify tab is still appearing on pending markets: http://localhost:3000/#/0xa776a48b4a62d57da6084c20858e31e654c829d0

**Comment by pimato** [comment]
> Looks like the verify tab is still appearing on pending markets: http://localhost:3000/#/0xa776a48b4a62d57da6084c20858e31e654c829d0

I don`t see it:
Bildschirmfoto 2020-11-13 um 20 47 56

**Comment by Mi-Lan** [comment]
> Looks like the verify tab is still appearing on pending markets: http://localhost:3000/#/0xa776a48b4a62d57da6084c20858e31e654c829d0

hmm i will check again but that shouldn't happen...

**Comment by Mi-Lan** [comment]
> Looks like the verify tab is still appearing on pending markets: http://localhost:3000/#/0xa776a48b4a62d57da6084c20858e31e654c829d0

@KadenZipfel i checked it and it should work so basically what i did instead is i evaluate if question.resolution.getTime() > currentTimestamp...

File: app/src/components/market/sections/market_navigation/index.tsx [hunk 1]
```diff
@@ -21,15 +23,18 @@ const MarketTab = styled.div<{ active: boolean }>`
...
   switchMarketTab: (arg0: string) => void
 }
 
 export const MarketNavigation = (props: Props) => {
-  const { activeTab, hasWinningOutcomes, isQuestionFinalized, resolutionDate, switchMarketTab } = props
+  const { activeTab, hasWinningOutcomes, marketMakerData, switchMarketTab } = props
+  const { isQuestionFinalized, question } = marketMakerData
+  const currentTimestamp = new Date().getTime()
+
+  const isOpen = question.resolution.getTime() > currentTimestamp
```
Comment (liaojikunwork) [inline app/src/components/market/sections/market_navigation/index.tsx:36]: It's not correct to check if it's opened market using resolution.getTime()

Please check openingTimestamp
https://github.com/protofire/omen-exchange/blob/bc7aa2babe31353b587e7c2b69523489ea26a54c/app/src/queries/markets_home.ts#L87

**Review by liaojikunwork — CHANGES_REQUESTED** [review]
See comments

File: app/src/components/market/sections/market_navigation/index.tsx (hunk 1 above)
Comment (Mi-Lan) [reply app/src/components/market/sections/market_navigation/index.tsx:36]: Screenshot 2020-11-17 at 13 08 28
should it then also be changed here? In market top details open. @KadenZipfel @liaojikunwork

**Review by liaojikunwork — APPROVED** [review]

**Review by pimato — APPROVED** [review]

File: app/src/components/market/sections/market_navigation/index.tsx (hunk 1 above)
Comment (KadenZipfel) [reply app/src/components/market/sections/market_navigation/index.tsx:36]: `question.resolution` should actually just be `openingTimestamp` converted to a `Date` object

**Comment by KadenZipfel** [comment]
I don't see any reason, based on code changes, to see why this issue would be directly related to this PR, but I noticed `market_verify` view is behaving unexpectedly. In the `verification` branch, this market which is apparently verified is displaying correctly, but in this PR the same market is not working the same.

`verification` branch:
Screen Shot 2020-11-17 at 10 22 39 AM

This PR:
Screen Shot 2020-11-17 at 10 20 15 AM

Can anyone else reproduce this or have any idea why this is going on?

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**